### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

This cleans up unused files and fixes CI.

## How can we measure success?

The codebase is easier to read

### CODE REVIEW

File: src/components/Image.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+const StyledImg = styled(Img)`
+  width: 80%;
+`
+
+export const Image: React.FC<ImageProps> = props => (
+  <StaticQuery
+    query={graphql`
+      query {
+        images: allFile(filter: { relativeDirectory: { glob: "images" } }) {
```
Comment (mxdvl) [inline src/components/Image.tsx:19]: Maybe select all images as described in the filesystem plugin?

File: src/content/home.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: 'Home'
+---
+
+<ContentSection>
+
+### We're shaping the future of digital journalism. <br/> [Join us](https://workforus.theguardian.com)
```
Comment (mxdvl) [inline src/content/home.mdx:8]: Markdown in components work!

Comment (buck06191) [reply src/content/home.mdx:8]: Interesting. Do we know how hard it is to style these? There's the ability to swap out default elements like `h1`, `h2`, etc., for custom elements that can be styled how we wish. 
https://www.gatsbyjs.com/docs/mdx/customizing-components/

Maybe this approach will be compatible with the markdown, meaning that we can set default styles for some of these?

**Comment by mxdvl** [comment]
Split the Image component into a separate PR: #19

**Review by buck06191 — APPROVED** [review]

**Pull request merged**